vendor/k8s.io/kubernetes/pkg/api/types.go

```diff
@@ -1256,6 +1256,9 @@ const (
 // Capability represent POSIX capabilities type
 type Capability string
 
+// CapabilityAll represent all POSIX capabilities types.
+var CapabilityAll Capability = "*"
```

Comment:
The comment should go on the SCC's `AllowedCapabilities` description so it shows up in generated documentation.